### PR TITLE
docs: Adds all possible values for JS agent

### PR DESCRIPTION
**Pull request by soluchok**
Adds all possible values for JS agent.

Signed-off-by: Andrii Soluk <isoluchok@gmail.com>

### CODE REVIEW

**Review by fqutishat — APPROVED** [review]

**Review by aholovko — APPROVED** [review]

File: cmd/agent-js-worker/README.md [hunk 1]
```diff
@@ -40,9 +40,29 @@ const agent = await new Agent.Framework({
...
+    "userConfig": {
+        "accessToken": "",
+        "walletSecretShare": ""
+    },
+    "useEDVCache": false,
+    "edvClearCache": "",
+    "useEDVBatch": false,
+    "edvBatchSize": 0,
+    "cacheSize": 0,
+    "opsKMSCapability": ""
```
Comment (llorllale) [inline cmd/agent-js-worker/README.md:65]: This will likely be removed very soon.

File: cmd/agent-js-worker/README.md [hunk 2]
```diff
@@ -40,9 +40,29 @@ const agent = await new Agent.Framework({
...
+    "edvCapability": "",
+    "blocDomain": "",
+    "trustbloc-resolver": "",
+    "authzKeyStoreURL": "",
+    "opsKeyStoreURL": "",
+    "edvOpsKIDURL": "",
+    "edvHMACKIDURL": "",
+    "kmsType": "",
+    "userConfig": {
+        "accessToken": "",
```
Comment (llorllale) [inline cmd/agent-js-worker/README.md:57]: This will likely be removed very soon.

**Comment by llorllale** [comment]
There are two options in there that I don't think are worth being added.

File: cmd/agent-js-worker/README.md (hunk 2 above)
Comment (soluchok) [reply cmd/agent-js-worker/README.md:57]: Ok, I am removing `accessToken` and `opsKMSCapability`

**Review by llorllale — APPROVED** [review]

**Pull request merged**